src/frontend/components/index.js

```diff
@@ -4,5 +4,8 @@ import { Sidebar} from "./sidebar/sidebar";
 import { SmallVideoCard } from "./smallVideoCard/smallVideoCard";
 import { CategoryBar } from "./categoryBar/categoryBar";
 import { SmallHorizontalVideoCard } from "./smallHorizontalVideoCard/smallHorizontalVideoCard"
+import { BigHorizontalCard} from "./bigHorizontalVideoCard/bigHorizontalVideoCard"
```

Comment:
![image](https://user-images.githubusercontent.com/75421090/184694955-58cb827e-6821-4d1d-9d5b-5c725336e90f.png)


aspect ratio of thumbnails seem to be off , do take care , in liked videos page